app/src/androidTest/java/com/example/henzoshimada/feeltrip/FeelTripTests.java

```diff
@@ -53,7 +53,7 @@ public void testUpdateQueueController(){
     }
 
     public void testAddMoodTask(){
-        ElasticSearchController.AddMoodTask addMoodTask = new ElasticSearchController.AddMoodTask();
+        ElasticSearchController.AddMoodTask addMoodTask = new ElasticSearchController.AddMoodTask(getActivity());
```

Comment:
Both AddMoodTask and EditMoodTask request that you send them context as a parameter now. Which context you choose to send doesn't really matter, since internally we call context.getApplicationContext()